R/calculate-pk-parameters-task.R

```diff
@@ -18,17 +18,47 @@ CalculatePKParametersTask <- R6::R6Class(
     generatedResultFileNames = NULL,
 
     #' @description
-    #' Create a `CalculatePKParametersTask` object
-    #' @param simulationFilePath simulation folder
-    #' @param simulationResultFilePaths simulation files
-    #' @param pkParametersToEvaluate list of PK parameters to evaluate
-    #' @param userDefinedPKFunctions list of user defined functions to calculate PK parameters
-    #' @param pkParameterResultsFilePath files where PK parameters are saved
-    #' @param generatedResultFileNames name of files where PK parameters are saved
-    #' @param ... parameters inherited from R6 class `Task` object
-    #' @return A new `CalculatePKParametersTask` object
-    initialize = function(...) {
-      super$initialize(...)
+    #' Save results from task run.
+    #' @param set R6 class `SimulationStructure`
+    #' @param taskResults list of results from task run.
+    saveResults = function(set,

```

Comment:
This identation is really messed up. 
Is there a reason why often you never put two parameter on the same line? 
The same question I also asked @abdullahhamadeh 
I understand that assignments are on multiple lines but this is just too much here

Can we just make sure that the function of a parameter call are on one line unless you go over the standard width of 120 col or whatever is the default and then you can align them? 

so for example 
```
 self$saveResults(
            set,
            taskResults
          )`
```
becomes

```
 self$saveResults(set,taskResults)
```